api/envoy/config/filter/http/grpc_stats/v2alpha/config.proto

```diff
@@ -20,6 +23,32 @@ message FilterConfig {
   // If true, the filter maintains a filter state object with the request and response message
   // counts.
   bool emit_filter_state = 1;
+
+  oneof per_method_stat_specifier {
+    // If set, specifies a whitelist of service/methods that will have individual stats
+    // emitted for them. Any call that does not match the whitelist will be counted
+    // in a stat with no method specifier: `cluster.<name>.grpc.*`.
+    api.v2.core.GrpcMethodList individual_method_stats_whitelist = 2;
+
+    // If set to true, emit stats for all service/method names.
+    //
+    // If set to false, emit stats for all service/message types to the same stats without including
```

Comment:
Is this now  the new default? Oneof can't distinguish an unset bool from a bool set to false.

Reply:
I think that's incorrect. I looked at the generated c++ from the proto, and the oneof has an enum of which type it is. One of the options is `PER_METHOD_STAT_SPECIFIER_NOT_SET`, so with that I can in fact tell the difference between `false` and `not specified`, but only because it's in a a `oneof`. I had it as a wrapped boolean before I realized I didn't need it wrapped.

Please correct me if my understanding of this is wrong.